src/panels/lovelace/card-features/hui-history-chart-card-feature.ts

```diff
@@ -185,9 +188,21 @@ class HuiHistoryChartCardFeature
       minX,
       maxX
     );
-    // add margin to the min and max
-    minY -= rangeY * 0.1;
-    maxY += rangeY * 0.1;
+    if (maxY < 0) {
+      // all values are negative
+      // add margin
+      maxY += rangeY * 0.1;
+      maxY = Math.min(0, maxY);
```

Comment:
These margins are not strictly necessary but they do make the chart look better so the fill doesn't completely disappear at the min or max points.